docs/dev_guide/example_gallery.rst

```diff
@@ -0,0 +1,20 @@
+.. _example_gallery:
+
+
+Example Gallery
+===============
+
+The purpose of the page is to describe the contribution guidelines for the `SunPy Example Gallery <http://docs.sunpy.org/en/stable/generated/gallery/index.html>`_. All potential contributors to the SunPy Example Gallery should read and abide by the following guidelines.
+
+Contribution Guidelines
+-----------------------
+
+-  The title of the example should be short yet descriptive and emphasize the goal of the example. Try to make the title appeal to a broad audience and avoid referencing a specific instrument, catalog, or anything wavelength dependent.
+-  Each example should begin with a paragraph that gives a brief overview of the entire example, including relevant astronomy concepts, and motivates the described functionality.
+-  The examples must be compatible with the versions supported by the last major release of the SunPy core package (i.e. Python >= 3.6).
+-  All the examples should be checked using one of the many PEP8 linters that are available (autopep8, pylint, flake8 as some examples).
```

Comment:
I would re-write this one to say something like, all the examples must be strictly PEP8 compliant. There are tools available to check this for you, i.e. flake8 ...